### ISSUE TITLE
[BUG] Неверное определение приоритета операторов встроенного языка

### ISSUE THREAD

**Issue by berlinsky-nk** [1c-syntax/bsl-language-server#3349]
**Версия**
Language 1C (BSL) v1.27.2 (VSCode)

**Описание ошибки диагностики**
Неверное определение приоритета операторов встроенного языка: `=` имеет больший приоритет, чем `Не`.
![image](https://github.com/user-attachments/assets/e0972a29-a325-4c84-be3e-d9428d67d2a8)

**Пример кода**
```
	Если Не БезопасныйРежим() = Ложь Тогда
		УстановитьОтключениеБезопасногоРежима(Истина);
	КонецЕсли;
```

**Скриншоты**
![image](https://github.com/user-attachments/assets/ac092f0d-70ec-40cd-abf7-50ce7fc61dfe)

**Дополнительная информация**
https://its.1c.ru/db/v836doc#bookmark:dev:TI000000146

**Comment by berlinsky-nk** [comment]
Исходя из формулировки замечания, диагностика как раз предлагает нам сравниваться с булево, что в примере и делается, а до знака равно она как будто даже не дошла, и до «ложь» тем более.

**Comment by berlinsky-nk** [comment]
Т.е. это не про «не сравнивайте булево с ложь/истина», а про «не используйте не_булево как логическое выражение»

**Comment by nixel2007** [comment]
Да, потому что диагностика не подозревает, что БезопасныйРежим может вернуть строку. приоритеты унарного "не" проверить тоже стоит. Но опять же, я не уверен, что там есть расчёт выражения, а не обычное шагание по дереву разбора/токенам
